tables/file.py

```diff
@@ -914,7 +914,7 @@ def create_group(self,
     def create_table(self,
                      where: Union[Group, str],
                      name: str,
-                     description: Optional[Description]=None,
+                     description: Union[dict, Type[IsDescription], Description, np.dtype, None]=None,
                      title: str="",
```

Comment:
Shouldn't we use [DTypeLike](https://numpy.org/devdocs/reference/typing.html#numpy.typing.DTypeLike) instead of `np.dtype`?

Reply:
Thanks @jtc42 
The criterion should be to have `DTypeLike` for parameters and `np.dtype` for outputs (unless there is a very good reason to do otherwise).
I would tend for fix `Tables.__init__` and other possible inconsistencies in this case.